etip/trackers/tests.py

```diff
@@ -321,3 +324,155 @@ def test_with_trackers(self):
             ]
         }
         self.assertJSONEqual(f.getvalue(), expected_json)
+
+
+EXODUS_API_BASE_URL = 'https://reports.exodus-privacy.eu.org'
```

Comment:
I would either put the constants at the beginning of the file (if we think it can be useful later one for other classes) or inside the class.

Reply:
Agreed, moved it!